tests/snmpsim/arbos.snmprec

```diff
@@ -0,0 +1,630 @@
+.1.3.6.1.2.1.1.1.0 = STRING: Peakflow SP 8.3 (build HIU6)  System Board Model: S2600CO Serial Number: CG22541072COI
```

Comment:
This is an snmpwalk, not an snmprec file.  The format is wrong.

That is what is causing tests to fail.

See https://docs.librenms.org/Developing/os/Test-Units/